backend/frontend/views.py

```diff
@@ -672,6 +672,44 @@ def action_stop_card(request, card_id):
     )
 
 
+@login_required()
+def view_partial_users_list(request):
+    exclude = {
+        "is_superuser": True,
```

Comment:
Why are you introducing a data structure for this filter but not for the ` active` filter? Also, please do display superusers.